src/mscorlib/shared/System.Private.CoreLib.Shared.projitems

```diff
@@ -557,7 +557,7 @@
   </ItemGroup>
   <ItemGroup Condition="$(TargetsWindows) and ('$(ClrProduct)' == 'core_clr' or '$(EnableWinRT)' == 'true')">
```

Comment:
'$(ClrProduct)' == 'core_clr' should not be needed with this change. This should merged into the ` '$(EnableWinRT)' == 'true'` block.

Reply:
Fixed